lightning/src/ln/channelmanager.rs

```diff
@@ -546,8 +546,10 @@ impl Into<u16> for FailureCode {
 struct MsgHandleErrInternal {
 	err: msgs::LightningError,
 	chan_id: Option<(ChannelId, u128)>, // If Some a channel of ours has been closed
-	shutdown_finish: Option<(ShutdownResult, Option<msgs::ChannelUpdate>)>,
+	shutdown_finish: Option<(ShutdownResult,
+	Option<msgs::ChannelUpdate>)>,
```

Comment:
nit: I believe we can revert this change to prevent unnecessary diff.

```suggestion
	shutdown_finish: Option<(ShutdownResult, Option<msgs::ChannelUpdate>)>,
```

Reply:
Dilemma of fixing obvious formatting vs. minimze diff :)
Accepted.